File: RadVolontera/RadVolontera.Models/FITPasos/FITPasos.cs
using RadVolontera.Models.Account;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RadVolontera.Models.FITPasos
{
    public class FITPasos
    {
        public long Id { get; set; }
        public bool IsValid { get; set; }
        public string UserId { get; set; }
        public DateTime DatumIzdavanja { get; set; }
        public UserResponse User { get; set; }
    }
}

File: RadVolontera/UI/radvolontera_mobile/lib/models/fit_pasos/fit_pasos.dart



import 'package:json_annotation/json_annotation.dart';
import 'package:radvolontera_mobile/models/account/account.dart';

part 'fit_pasos.g.dart';
@JsonSerializable()
class FITPasosModel {
  int? id;
 DateTime ? datumIzdavanja;
 bool isValid;
 String? userId;
AccountModel? user;
 
FITPasosModel(this.id, this.datumIzdavanja, this.isValid,this.user, this.userId);
      /// A necessary factory constructor for creating a new User instance
  /// from a map. Pass the map to the generated `_$UserFromJson()` constructor.
  /// The constructor is named after the source class, in this case, User.
   factory FITPasosModel.fromJson(Map<String, dynamic> json) => _$FITPasosModelFromJson(json);

  // /// `toJson` is the convention for a class to declare support for serialization
  // /// to JSON. The implementation simply calls the private, generated
  // /// helper method `_$UserToJson`.
   Map<String, dynamic> toJson() => _$FITPasosModelToJson(this);
}